### DISCUSSION TITLE
Kontaktaufnahme zu Steffeeeen1234

### DISCUSSION THREAD

**Discussion by andygrunwald** [andygrunwald/FOM-LaTeX-Template#193] [General]
Hey @Steffeeeen1234,
leider konnte ich deine E-Mail-Adresse nicht finden. Wäre schön, wenn du dich mal bei mir (andygrunwald [at] g m a i l [dot] com) melden könntest.

Danke dir :)

**Comment by Steffeeeen1234** [comment]
Habe jetzt auch meine E-Mail Settings bei guthub entdeckt!

**Reply by andygrunwald** [reply]
Top. Danke :) (auch für die Kontaktaufnahme)
